ksp_plugin/plugin.cpp

```diff
@@ -855,7 +855,7 @@ void Plugin::CreateFlightPlan(GUID const& vessel_guid,
   FindOrDie(vessels_, vessel_guid)->CreateFlightPlan(
       final_time,
       initial_mass,
-      prediction_parameters_,
+      DefaultPredictionParameters(),
```

Comment:
We could use the prediction parameters of the relevant vessel, but then we would need to construct corresponding burn parameters; since this is consistent with the existing (unintended) behaviour, OK with punting on it.